lib/websocket/utils/result.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
```

Comment:
Should be moved to metautil

Reply:
Should I move the Result class to metautil, or do you think it would be better to introduce an algebraic data types?